admin_guide/assembly_restoring-cluster.adoc

```diff
@@ -30,6 +30,7 @@ include::admin_guide/topics/proc_restoring-cluster.adoc[leveloffset=+1]
 
 include::day_two_guide/topics/proc_restoring-master.adoc[leveloffset=+1]
 
+
```

Comment:
If the static pod's restored along with the masters, this topic might need rework.